src/mobile-token/MobileTokenContext.tsx

```diff
@@ -193,8 +206,7 @@ export const MobileTokenContextProvider: React.FC = ({children}) => {
       const cancelTimeoutHandler = timeoutHandler(() => {
         // When timeout has occured, we notify errors in Bugsnag
         // and set state that indicates timeout.
-        setTimeoutFlag(true);
-        setIsLoading(false);
+        setStatus('timeouted');
```

Comment:
I think it's spelled "timed-out", like "something has timed out"?
Ref. https://www.oxfordlearnersdictionaries.com/definition/american_english/time-out